cmake/modules/FindUFE.cmake

```diff
@@ -51,13 +51,19 @@ if(UFE_INCLUDE_DIR AND EXISTS "${UFE_INCLUDE_DIR}/ufe/ufe.h")
     if(UFE_MAJOR_VERSION VERSION_EQUAL "0")
         math(EXPR UFE_PREVIEW_VERSION_NUM "${UFE_MINOR_VERSION} * 1000 + ${UFE_PATCH_LEVEL}")
     elseif(UFE_VERSION VERSION_EQUAL "4.1.0")
-        # Temporary. Once next Maya PR is released with UFE v4.1.0 this should
+        # Temporary. Once next Maya is released with UFE v4.1.0 this should
         # be removed (along with all the UFE_PREVIEW_VERSION_NUM checks).
         set(UFE_PREVIEW_VERSION_NUM 4100)
-    elseif((UFE_VERSION VERSION_GREATER_EQUAL "4.0.100") AND (UFE_VERSION VERSION_LESS "5.0.0"))
-        # Temporary. Once next Maya PR is released with UFE v4.X.0 this should
-        # be changed into UFE_PREVIEW_VERSION_NUM of 4XYY, where YY is the last value from "4.0.YY"
-        math(EXPR UFE_PREVIEW_VERSION_NUM "4 * 1000 + ${UFE_MINOR_VERSION} * 100 + ${UFE_PATCH_LEVEL}")
+    elseif(UFE_VERSION VERSION_EQUAL "4.2.0")
+        # Temporary. Once next Maya is released with UFE v4.2.0 this should
+        # be removed (along with all the UFE_PREVIEW_VERSION_NUM checks).
+        set(UFE_PREVIEW_VERSION_NUM 4202)
```

Comment:
Set at 4202 since that is the last value (4.1.202) it was before being made final (4.2.0).